.github/workflows/github_issues.yml

```diff
@@ -0,0 +1,22 @@
+name: Sync issue to Azure DevOps work item
+
+"on":
+  issues:
+    types:
+      [opened, edited, deleted, closed, reopened, labeled, unlabeled]
```

Comment:
I did not locate modelid changes. Did I miss it?

Reply:
This PR does not have the model ID changes. I did not want to introduce the API version changes in the PR, so I will create a new PR once this is in.